src/controllers/notifications.js

```diff
@@ -70,7 +71,19 @@ const model = require('../models/notifications')
 exports.post = function * postNotification () {
   const notification = yield requestUtil.validateBody(this, 'Notification')
   log.debug('Got notification: ' + JSON.stringify(notification))
-  yield model.processNotification(notification)
-
+  try {
+    yield model.processNotification(notification)
+  } catch (e) {
+    // If this receives UnacceptableExpiryError, do not respond with HTML error 422.
+    // Doing so would cause Ledger to reply with the same message, creating an infinite loop.
+    if (e instanceof UnacceptableExpiryError) {
```

Comment:
Why is the model throwing an exception if it should be ignored? It probably makes sense to put this business logic into the model